docs/pages/experiments/base/components-gallery.tsx

```diff
@@ -115,8 +157,142 @@ export default function ComponentsGallery() {
+  };
+
+  const handleCopyOnEnter = () => {
+    setCopySnackbarExited(false);
+  };
+
+  const handleCopyOnExited = () => {
+    setCopySnackbarExited(true);
+  };
+
+  const [settingsAnchor, setSettingsAnchor] = React.useState<null | HTMLElement>(null);
+
+  const settingsButtonHandleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setSettingsAnchor(settingsAnchor ? null : event.currentTarget);
+  };
+
+  const settingsOpen = Boolean(settingsAnchor);
+  const settingsId = settingsOpen ? 'settings-popup' : undefined;
+
+  const colorPickerSliderChangeHandler = (e: Event, value: number | number[]) => {
+    setRootStyles(`
+:root {
+  --primary-50: hsl(${value}, 76%, 95%);
+  --primary-100: hsl(${value}, 70%, 86%);
+  --primary-200: hsl(${value}, 59%, 75%);
+  --primary-300: hsl(${value}, 49%, 60%);
+  --primary-400: hsl(${value}, 68%, 38%);
+  --primary-500: hsl(${value}, 76%, 21%);
+  --primary-600: hsl(${value}, 80%, 18%);
+  --primary-700: hsl(${value}, 84%, 15%);
+  --primary-800: hsl(${value}, 86%, 12%);
+  --primary-900: hsl(${value}, 90%, 8%);
+}
+    `);
+
+    const styleTag =
+      document.getElementById('gallery-overrides') ?? document.createElement('style');
+    styleTag.id = 'gallery-overrides';
+    styleTag.innerHTML = rootStyles;
+    document.getElementsByTagName('head')[0].appendChild(styleTag);
+  };
+
   return (
-    <Stack gap={2} style={{ padding: '8px' }}>
+    <Stack gap={2} sx={{ padding: 1 }}>
+      {/* Copy theme button */}
+      <Box sx={{ position: 'absolute', right: '10px' }}>
+        <CopyButton className="GalleryButton" style={{ float: 'right' }} onClick={copyTheme}>
+          Export theme
+        </CopyButton>
+        <Snackbar
+          autoHideDuration={2000}
+          open={copySnackbarOpen}
+          onClose={handleCopyClose}
+          exited={copySnackbarExited}
+          className="GallerySnackbar"
+        >
+          <Transition
+            timeout={{ enter: 400, exit: 400 }}
+            in={copySnackbarOpen}
+            appear
+            unmountOnExit
+            onEnter={handleCopyOnEnter}
+            onExited={handleCopyOnExited}
+            nodeRef={copyNodeRef}
+          >
+            {(status) => (
+              <div
+                className="GallerySnackbar-content"
+                style={{
+                  transform: positioningStyles[status],
+                  transition: 'transform 300ms ease',
+                }}
+                ref={copyNodeRef}
+              >
+                <CheckRoundedIcon
+                  sx={{
+                    color: 'success.main',
+                    flexShrink: 0,
+                    width: '1.25rem',
+                    height: '1.5rem',
+                  }}
+                />
+                <div className="snackbar-message">
+                  <p className="snackbar-title">Done!</p>
+                  <p className="snackbar-description">The theme stylesheet has been copied!</p>
```

Comment:
To be a bit more explicit:

```suggestion
                  <p className="snackbar-description">The theme stylesheet has been copied to clipboard!</p>
```